src/AppBundle/Utils/Validator.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Utils;
+
+class Validator
+{
+    public function username($username)
```

Comment:
@elkuku, You right, but a method name in most cases should begin with a verb. :)